pkg/tests/tasks/traffic/ingress/yaml_vars.go

```diff
@@ -38,6 +38,11 @@ var (
 	meshNamespace = env.GetDefaultMeshNamespace()
 )
 
+var (
+	// OCP4.x
+	SMCP_NAME = env.GetDefaultSMCPName()
```

Comment:
Why don't you inline this? We don't need the variable (and its name is also not formatted correctly).

Reply:
i have updated it.